src/EnergyPlus/DXCoils.cc

```diff
@@ -231,7 +231,7 @@ void SimDXCoil(EnergyPlusData &state,
         CalcDXHeatingCoil(state, DXCoilNum, PartLoadRatio, fanOp, AirFlowRatio, MaxCap);
     } break;
     case HVAC::CoilVRF_FluidTCtrl_Cooling: {
-        CalcVRFCoolingCoil_FluidTCtrl(state, DXCoilNum, HVAC::CompressorOp::On, FirstHVACIteration, PartLoadRatio, fanOp, CompCycRatio, _, _);
+        CalcVRFCoolingCoil_FluidTCtrl(state, DXCoilNum, HVAC::CompressorOp::On, FirstHVACIteration, PartLoadRatio, fanOp, CompCycRatio, _, _, MaxCap);
```

Comment:
I'm not sure why the max capacity is not always passed into the coil. It's either 1.0e+20 or it's a value. That's for another day.

Reply:
As the parent function `SimDXCoil` has it as an optional argument, its not always available, so I made it an optional arg as well for the `CalcVRFCoolingCoil_FluidTCtrl` called inside `SimDXCoil`.